float64[] temperature_list
float64[] pressure_list
float64[] altitude_list
bool connection_status
string[] sensor_names